barbarik.py

```diff
@@ -834,11 +833,7 @@ def one_experiment(self, experiment, j, i, numExperiments, tj):
                 i += 1
                 ok, breakExperiment = exp.one_experiment(experiment, j, i, numExperiments, tj)
 
-                if ok is None:
```

Comment:
Again, this is likey not OK, there is a valid `None, None` return here? Or am I missing something?

Also, this will collide with the other merge request you made, right?